server/common/transport-shared/src/main/java/org/kaaproject/kaa/server/transport/message/AbstractMessage.java

```diff
@@ -13,26 +13,32 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.kaaproject.kaa.server.operations.service.akka.messages.io.request;
-
-import io.netty.channel.ChannelHandlerContext;
+package org.kaaproject.kaa.server.transport.message;
 
 import java.util.UUID;
 
-import org.kaaproject.kaa.server.operations.service.akka.messages.io.PlatformAware;
-import org.kaaproject.kaa.server.operations.service.http.commands.ChannelType;
+import org.kaaproject.kaa.server.transport.channel.ChannelContext;
+import org.kaaproject.kaa.server.transport.channel.ChannelType;
+import org.kaaproject.kaa.server.transport.platform.PlatformAware;
 
-public class AbstractRequestMessage implements PlatformAware{
+/**
+ * Abstract class to represent platform aware message. Holds references to
```

Comment:
An abstract class that represents a platform aware message. It includes references to response and error converters that are used to convert objects into the channel specific data.